com.yubico.yubioath.yml

```diff
@@ -43,127 +42,19 @@ modules:
+  - name: yubioath-desktop
     buildsystem: simple
     build-commands:
-      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
-        --prefix=${FLATPAK_DEST} "cryptography" --no-build-isolation --no-binary cryptography
-    sources:
-      - type: file
-        url: https://files.pythonhosted.org/packages/5e/0b/95d387f5f4433cb0f53ff7ad859bd2c6051051cebbb564f139a999ab46de/pycparser-2.21.tar.gz
-        sha256: e644fdec12f7872f86c58ff790da456218b10f863970249516d60a5eaca77206
-        x-checker-data:
-          type: pypi
-          name: pycparser
-      - type: file
-        url: https://files.pythonhosted.org/packages/2b/a8/050ab4f0c3d4c1b8aaa805f70e26e84d0e27004907c5b8ecc1d31815f92a/cffi-1.15.1.tar.gz
-        sha256: d400bfb9a37b1351253cb402671cea7e89bdecc294e8016a707f6d1d8ac934f9
-        x-checker-data:
-          type: pypi
-          name: cffi
-      # cyrptography 35 is too new
-      - type: file
-        url: https://files.pythonhosted.org/packages/cc/98/8a258ab4787e6f835d350639792527d2eb7946ff9fc0caca9c3f4cf5dcfe/cryptography-3.4.8.tar.gz
-        sha256: 94cc5ed4ceaefcbe5bf38c8fba6a21fc1d365bb8fb826ea1688e3370b2e24a1c
-      - pycrypto.json # regenrate on update
-  - python3-yubikey-manager.json
-  - name: yubioath-desktop
-    buildsystem: qmake
-    post-install:
-      - install -Dm644 resources/com.yubico.yubioath.desktop /app/share/applications/com.yubico.yubioath.desktop
-      - install -Dm644 resources/icons/com.yubico.yubioath.svg /app/share/icons/hicolor/scalable/apps/com.yubico.yubioath.svg
-      - install -Dm644 resources/com.yubico.yubioath.appdata.xml /app/share/appdata/com.yubico.yubioath.appdata.xml
+      - mkdir /app/yubioath-desktop
+      - cp -r * /app/yubioath-desktop
+      - mkdir -p /app/bin
+      - ln -s /app/yubioath-desktop/authenticator /app/bin/authenticator
+      - mkdir -p /app/share/applications
+      - cp /app/yubioath-desktop/com.yubico.authenticator.desktop /app/share/applications/com.yubico.yubioath.desktop
+      - mkdir -p /app/share/icons/hicolor/128x128/apps
+      - cp /app/yubioath-desktop/com.yubico.yubioath.png /app/share/icons/hicolor/128x128/apps/com.yubico.yubioath.png
     sources:
       - type: archive
-        url: https://developers.yubico.com/yubioath-desktop/Releases/yubioath-desktop-5.1.0.tar.gz
-        sha256: 2e6f45e1e686f53e5bac057b5c3024a23d160a5997118221b94ce1dab934a1cd
-        x-checker-data:
-          type: rotating-url
-          url: https://developers.yubico.com/yubioath-desktop/Releases/yubioath-desktop-latest.tar.gz
-          pattern: https://developers.yubico.com/yubioath-desktop/Releases/yubioath-desktop-([0-9.]+).tar.gz
-      - type: patch
-        paths:
-          - yubioath-desktop-path.patch
-          - yubioath-desktop-releases.patch
-          - back-port-install-fix.patch
-          - fix-appdata.patch
+        url: https://developers.yubico.com/yubioath-desktop/Beta/yubioath-desktop-6.0.0-beta.1-linux.tar.gz
```

Comment:
Why you are switching from source build to fetching binary?

Reply:
from what i can find, there doesn't seem to be any public repo or branch for the beta ver

Reply:
As someone who's worked on a source build there just to many dependencies and overall i've been looking int moving the master branch to binaries as well especially since a lot of deps like openssl and pycrptography have to be built in specific  orders or things just stop working

Reply:
Hm, so the app is not open source anymore?

Reply:
Not sure.  Therr is no info from upstream

Reply:
I think 5.1.0 should be kept with source build since it works now.

Reply:
We are looking to merge this into beta not stable

Reply:
I know, this is 6.0. You mentioned moving master to binary though.

Reply:
Master is broken currently for the majority of people. Hence the discussion.  I'd rather ship binaries over something that doesn't work

Reply:
But this one [broken](https://github.com/flathub/com.yubico.yubioath/pull/59?notification_referrer_id=NT_kwDOAjzamLM0NjA3MjM0NzU3OjM3NTQyNTUy&notifications_query=is%3Aunread#issuecomment-1279184674) the same,no? The nature of fedora bug makes it unfixable whatever you do in flatpak.

Reply:
yea the issue is still there, i added a disclaimer to the appstream description in this pr that will at least tell users that there's a problem with the flatpak